package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/busboy",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
why the bump?

Reply:
The bump came from the current/old `next` branch